controller/recurring_job_controller.go

```diff
@@ -377,6 +381,11 @@ func (control *RecurringJobController) newCronJob(recurringJob *longhorn.Recurri
 							Tolerations:        util.GetDistinctTolerations(tolerations),
 							NodeSelector:       nodeSelector,
 							PriorityClassName:  priorityClass.Value,
+							ImagePullSecrets: []corev1.LocalObjectReference{
+								{
+									Name: registrySecretSetting.Value,
```

Comment:
If the setting is empty, what happens? should we ignore the empty case here?

Reply:
Yes, we should. Thanks for the reminder.